lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -1721,68 +1701,117 @@ def cmd_db_status(*args)
     return if not db_check_driver
 
     if framework.db.connection_established?
-      cdb = ''
-      ::ActiveRecord::Base.connection_pool.with_connection do |conn|
-        if conn.respond_to?(:current_database)
-          cdb = conn.current_database
-        end
-      end
-      print_status("#{framework.db.driver} connected to #{cdb}")
+      print_connection_info
     else
       print_status("#{framework.db.driver} selected, no connection")
     end
   end
 
   def cmd_db_connect_help
-    # Help is specific to each driver
-    cmd_db_connect("-h")
+    print_line("   Usage: db_connect <user:pass>@<host:port>/<database>")
+    print_line("      OR: db_connect -y [path/to/database.yml]")
+    print_line("      OR: db_connect <options> <http|https>://<host:port>")
```

Comment:
Minor: consider changing `<options>` to `[options]` to indicate that it is optional rather than required